src/options/views/edit/settings.vue

```diff
@@ -6,6 +6,14 @@
         <input type="checkbox" v-model="config.shouldUpdate">
         <span v-text="i18n('labelAllowUpdate')"></span>
       </label>
+      <label class="flex">
+        <span v-text="i18n('labelNotifyUpdates')" class="mr-1"/>
+        <select v-model="config.notifyUpdates">
+          <option value="" v-text="i18n('useGlobalSetting')"/>
+          <option value="1" v-text="i18n('buttonEnable')"/>
+          <option value="0" v-text="i18n('buttonDisable')"/>
```

Comment:
`buttonEnable` and `buttonDisable` were supposed to be two actions, or verbs, I am not sure if they are proper here. Maybe it should be `enabled` and `disabled`?

@tophf What's your point of view?

Reply:
Currently the choices are:

* Use global setting
* Enable
* Disable

All three are verbs so I don't think it's weird. For example the old option "Allow update" is also an imperative verb.

Reply:
OK, get it.

Reply:
If an opportunity presents itself like another similar option, we can add `On`, `Off` or something generic of the kind. As for `Use global setting`, right now I don't see how to convert it to a noun-based state description while keeping it short and self-evident, but that's a problem for the future.

Reply:
@gera2ld hmm you got me thinking and now I'm not sure... What about this:

[x] Allow update + [notify on update | don't notify on update | use global notification mode]

Each string of these three will be an entire option inside the selector.
The plus sign is a hardcoded text.

Reply:
I prefer the action + status version, i.e. `Notify on update: [true | false | (default)]`, because the options are shorter and easier to read in my opinion.

Reply:
What **exactly** does `(default)` do from a user's viewpoint? It's too vague.

Reply:
```
[x] Allow update
[x] Notify on update or [x] use the global option in settings
```

The notification checkbox is automatically disabled when the global is checked.
Mm?

Reply:
Another idea is a radio button:

```
Allow update [x]
Update notification: ( ) on ( ) off (x) use global setting
```

Reply:
What's the difference between radio button and select box?
I think the last one looks good for me, because the options are simple enough so that I don't have to think much about what each option means behind the words.

Reply:
The differences are:

* all the options are immediately visible
* there's no long empty rectangle when a shorter option is selected
* one click less to switch

Reply:
Sounds great. Much better than hierarchical options, like one option depends on or affects another.